Src/Common/FSecure/C3/Interfaces/Connectors/Covenant.cpp

```diff
@@ -92,7 +92,7 @@ namespace FSecure::C3::Interfaces::Connectors
 		/// @param jitter percent to jitter the delay by
 		/// @param listenerId the id of the Bridge listener for covenant
 		/// @return generated payload.
-		FSecure::ByteVector GeneratePayload(ByteView binderId, std::string manualExecution, std::string pipename, uint32_t delay, uint32_t jitter, uint32_t connectAttempts);
+		FSecure::ByteVector GeneratePayload(ByteView binderId, std::string manualExecution, std::string gruntType, std::string pipename, uint32_t delay, uint32_t jitter, uint32_t connectAttempts);
```

Comment:
This is how it done in the whole project..
For now I prefer to leave it like this, because it will take us a lot of time to change it all.
We'll do it as the next phase.